thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -2255,8 +2255,10 @@ class CLookupJoinActivityBase : public CInMemJoinBase<HTHELPER, IHThorHashJoinAr
             }
             virtual bool spill(bool critical) // called from OOM callback
             {
+                if (!channelCollector->spill(critical))
+                    return false;
```

Comment:
NB: This is an incidental fix/change.
Changing when the 'split' flag is set, to after the spill is done.
Previously changing it before, could mean it was picked up before spill occurred and shink was called with little effect.
Though doing so did not cause any particular problems, other than not freeing memory (row array memory) as soon as it could.